core/api/src/main/java/edu/uci/ics/texera/api/schema/Schema.java

```diff
@@ -108,6 +108,20 @@ public String toString() {
         return "Schema[" + this.attributes.toString() + "]";
     }
     
+    public static boolean isAttributeType(Schema schema, String attributeName, AttributeType type)

```

Comment:
To me this new helper function seems redundant. See comment below where this function is used.

Reply:
agreed. Removed and modified the usage in calling place